### PR TITLE
FEATURE | HEAD-1733 | Sub Nav Link Mapping

### PR DESCRIPTION
**Pull request by CoderQuest**

### CODE REVIEW

File: src/classes/CollectionSubnavSettings.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import collectionSubnavSettingsJson from '../templates/collection-subnav-settings.json';
```
Comment (CoderQuest) [inline src/classes/CollectionSubnavSettings.ts:1]: This is new

File: src/templates/collection-subnav-settings.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
```
Comment (CoderQuest) [inline src/templates/collection-subnav-settings.json:1]: These are new

File: src/templates/option-fields.json [hunk 3]
```diff
@@ -15,6 +15,10 @@
     {
       "id": "gwpSettings",
       "value": "GWP settings"
+    },
```
Comment (CoderQuest) [inline src/templates/option-fields.json:18]: new here

**Review by Kyon147 — APPROVED** [review]
LGTVM! Great work Yi. 

I'll sort out the jslint at some point so the styling matches the other repo.

**Pull request merged**